Data/OJS.Data.Models/Problems/ProblemResource.cs

```diff
@@ -27,5 +29,11 @@ public class ProblemResource : DeletableAuditInfoEntity<int>, IOrderableEntity
         public string? Link { get; set; }
 
         public double OrderBy { get; set; }
+
+        public void RegisterMappings(IProfileExpression configuration)
+            => configuration.CreateMap<ProblemResource, ProblemResource>()
+                .ForMember(
+                    d => d.Id,
+                    opt => opt.MapFrom(src => 0));
```

Comment:
This should not be here. It is not mapping logic, but business operation logic. Also entity models should be kept clean from mapping configurations.